cmd/edit.go

```diff
@@ -112,6 +112,15 @@ var EditSecretCmd = &cobra.Command{
 
 			utils.Check(err)
 
+			if !secret.Editable() {
+				notice := []byte(`
+# DANGER! Secrets cannot be updated, only replaced. Once the change is applied, the old values will be lost forever.
+# Note: You can exit without saving to skip.
+			
```

Comment:
These extra spaces/tabs make this an invalid YAML file

Reply:
Not sure about the "DANGER!" part too. Maybe just "WARNING!" like the other one?